src/components/MermaidDash/components/LoadingIndicator.jsx

```diff
@@ -49,24 +49,25 @@ const StyledHeader = styled.header`
   `)}
 `
 
-const LoadingIndicator = ({ projectData, showLoadingIndicator, setShowLoadingIndicator }) => {
+const LoadingIndicator = ({ loadedProjectsCount, totalProjectsCount, isApiDataDoneLoading }) => {
```

Comment:
Oh that is a very good call. Will you let me do this on a different branch to avoid merge conflicts? (Ive got a series of branches branched of this one and its descendants.)